improved polish I18n

bigoss opened the pull request.

vitalidze commented:
Why have you closed the pull request?

Also I can see whitespaces changes in your files, why have you applied them?

bigoss commented:
Hi .
Didn'close i mean did not intentionaly . Probably Forgot to remove spaces sorry
Wił remove them push and send pul request again

Kind regards 

Jaroslaw
Sent from my iPhone

> On 8 mar 2015, at 10:34, Vitaly Litvak notifications@github.com wrote:
> 
> Why have you closed the pull request?
> 
> Also I can see whitespaces changes in your files, why have you applied them?
> 
> —
> Reply to this email directly or view it on GitHub.

bigoss commented:
Hi Vitaly,

I have sent pull request again and changed whitecaps hope they look as it was b4
Sorry for messing I am not familiar with using intellij and github so mistakes may happen .
Translation works for sure is all copied from another language file and replaced with polish words.
The original polish translation was less usable , many things were not included in  .pl file.

Best Regards

Jaroslaw Boss 

> On 08 Mar 2015, at 10:34, Vitaly Litvak notifications@github.com wrote:
> 
> Why have you closed the pull request?
> 
> Also I can see whitespaces changes in your files, why have you applied them?
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/vitalidze/traccar-web/pull/105#issuecomment-77740894.
